src/arrayMethodSort.js

```diff
@@ -4,8 +4,23 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
```

Comment:
Avoid extending the Array.prototype. It's considered bad practice because it can cause unexpected side effects with other code or libraries that may also be trying to extend the Array.prototype with the same method name.

Reply:
https://mate.academy/chats/27190?selectedMessageId=1401459